### PR TITLE
Feature Request: Show reminder even when ring volume is not muted

### PR DESCRIPTION
**Pull request by yoobi**
Adding requested feature #24

### CODE REVIEW

File: app/src/main/java/com/github/muellerma/mute_reminder/MediaAudioManager.kt [hunk 1]
```diff
@@ -9,11 +9,14 @@ import androidx.core.content.getSystemService
...
 class MediaAudioManager(context: Context) {
     private val audioManager: AudioManager = context.getSystemService()!!
+    private val prefs = Prefs(context)
 
     fun shouldNotify(): Boolean {
-        return isRingToneMuted() && !isMediaMuted() && !usesRemoteOutput()
+        return (isRingToneMuted() || isRingToneIgnored()) && !isMediaMuted() && !usesRemoteOutput()
     }
 
+    private fun isRingToneIgnored() = prefs.ignoreRingTone
```
Comment (mueller-ma) [inline app/src/main/java/com/github/muellerma/mute_reminder/MediaAudioManager.kt:18]: I'd drop this function.

**Review by mueller-ma — COMMENTED** [review]
I review this again when #28 is merged.

File: app/src/main/java/com/github/muellerma/mute_reminder/MediaAudioManager.kt (hunk 1 above)
Comment (yoobi) [reply app/src/main/java/com/github/muellerma/mute_reminder/MediaAudioManager.kt:18]: I thought it was cleaner to do like this as all the condition are functions, I can remove it tho if you prefer

**Comment by mueller-ma** [comment]
Can you rebase this on `main`?

**Comment by yoobi** [comment]
Is everything okay with my merge ?

**Comment by mueller-ma** [comment]
Can you look at my latest commits here? Maybe there's a better name than `Notify only when ring tone is muted`.

**Comment by yoobi** [comment]
I couldn't find a better name, we can add something like "Notify when ring tone is muted but media is ON" however I don't think this is really better. Plus users who have this app already know the purpose of this app

**Comment by yoobi** [comment]
While coming with a better name, can this be merged ?

**Pull request merged**